.github/workflows/CIGeoInterfaceMakie.yml

```diff
@@ -0,0 +1,42 @@
+name: CI
+on:
+  push:
+    branches:
+      - main
+    tags: ['*']
+  pull_request:
+concurrency:
+  # Skip intermediate builds: always.
+  # Cancel intermediate builds: only if it is a pull request build.
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: ${{ startsWith(github.ref, 'refs/pull/') }}
+jobs:
+  test:
+    name: Julia ${{ matrix.version }} - ${{ matrix.os }} - ${{ matrix.arch }} - ${{ github.event_name }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        version:
+          - '1.7'
+          - '1'
+          - 'nightly'
+        os:
+          - ubuntu-latest
+        arch:
+          - x64
+    steps:
+      - uses: actions/checkout@v2
+      - uses: julia-actions/setup-julia@v1
+        with:
+          version: ${{ matrix.version }}
+          arch: ${{ matrix.arch }}
+      - uses: julia-actions/cache@v1
+      - uses: julia-actions/julia-buildpkg@v1
+      - name: Run julia tests
+        shell: julia --project=GeoInterfaceMakie
+        run: |
```

Comment:
This looks a bit hacky. Is this the right way to do CI for subpackages?

Reply:
Hah I don't even know, I just realised we probably aren't testing the Plots.jl interface eithere

Reply:
Oh no its actually here:
https://github.com/JuliaGeo/GeoInterface.jl/blob/main/.github/workflows/recipes.yml

Probably copy that

Reply:
It seems that file never ran, hopefully I fixed that.